Dockerfile

```
@@ -90,22 +102,40 @@ RUN mkdir -p /etc/apt/keyrings \
   && apt-get clean \
   && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-ARG TYPOS_VERSION=1.14.6
+# todo track with updatecli (with cst.yml)
+# add platform aarch64 compliance
+ARG TYPOS_VERSION=1.32.0
 # ARG TYPOS_SHASUM_256="27ce43632f09d5dbeb2231fe6bbd7e99eef4ed06a9149cd843d35f70a798058c"
-RUN curl --silent --show-error --location --output /tmp/typos.tar.gz \
-  "https://github.com/crate-ci/typos/releases/download/v${TYPOS_VERSION}/typos-v${TYPOS_VERSION}-x86_64-unknown-linux-musl.tar.gz" \
+RUN ARCH="$(uname -m)" && \
+  if [ "$ARCH" = "x86_64" ]; then \
+    DOWNLOAD_URL="https://github.com/crate-ci/typos/releases/download/v${TYPOS_VERSION}/typos-v${TYPOS_VERSION}-x86_64-unknown-linux-musl.tar.gz"; \
+    elif [ "$ARCH" = "aarch64" ] || [ "$ARCH" = "arm64" ]; then \
+    DOWNLOAD_URL="https://github.com/crate-ci/typos/releases/download/v${TYPOS_VERSION}/typos-v${TYPOS_VERSION}-aarch64-unknown-linux-musl.tar.gz"; \
+  else \
+    echo "Unsupported architecture: $ARCH" && exit 1; \
+  fi \
+  && curl --silent --show-error --location --output /tmp/typos.tar.gz "${DOWNLOAD_URL}" \
```

Easier to rely on the output of `uname` in this case:

```suggestion
RUN curl --silent --show-error --location --output /tmp/typos.tar.gz \
  "https://github.com/crate-ci/typos/releases/download/v${TYPOS_VERSION}/typos-v${TYPOS_VERSION}-$(uname -m)-unknown-linux-musl.tar.gz" \
```

Example on our productions:

```bash
dduportal@ci.jenkins.io:~$ uname -m
aarch64

dduportal@archives:~$ uname -m
x86_64
```